cardano-node/src/Cardano/Tracing/OrphanInstances/Shelley.hs

```diff
@@ -1018,20 +1018,20 @@ instance ToJSON (Alonzo.CollectError StandardCrypto) where
         object
           [ "kind" .= String "PlutusTranslationError"
           , "error" .= case err of
-              Alonzo.ByronInputInContext -> String "Byron input in the presence of a plutus script"
-              Alonzo.ByronOutputInContext -> String "Byron output in the presence of a plutus script"
-              Alonzo.TranslationLogicErrorInput -> String "Logic error translating inputs"
+              Alonzo.ByronTxOutInContext txOutSource ->
+                String $ "Byron output in the presence of a plutus script, output source: " <> show txOutSource
+              Alonzo.TranslationLogicMissingInput txin ->
+                String $ "Logic error translating input: " <> show txin
```

Comment:
What does "Logic error" mean?

Reply:
We can come up with a better name for the type in a separate ledger PR, and change the prose here. The error indicates that a transaction input does not exist in the UTxO.

In the ledger code, this is indeed a logic error, since we only call the translation function after we have checked that the inputs are valid. So it's a bug if it ever fires. Since we've re-used the translation for the CLI, however, it can happen not as a true logic error. Great catch!

Reply:
So we can ignore this for now and it will be removed/renamed in a future PR in ledger?

Reply:
> that a transaction input does not exist in the UTxO

Why not "Transaction input does not exist in the UTxO"

Reply:
works for me!